monitoring/tests/unit/test__dataframe.py

```diff
@@ -14,14 +14,13 @@
 
 try:
     import pandas
-except ImportError:
+except ImportError:  # pragma: NO COVER
```

Comment:
At least one of these two branches should happen in our normal tests. I would assume that the `HAVE_PANDAS = True` branch is the one we want to ensure is covered.